src/main/java/TaskManager.java

```diff
@@ -0,0 +1,82 @@
+public class TaskManager {
+    private Task[] tasks;
+    private int count;
+    private boolean isTaskAdded;
+
+    public TaskManager(int capacity) {
+        tasks = new Task[capacity];
+        this.count = 0;
+        this.isTaskAdded = false;
+    }
+
+    public int getCount() {
```

Comment:
```suggestion
    public int size() {
```
The method name getCount() doesn’t follow Java’s collection naming conventions.